README.rst

```diff
@@ -23,7 +23,11 @@ Development
     # Install for development
     bv bash
     cd deep_folding
-    python setup.py develop
+    virtualenv -p /casa/install/bin/python venv
```

Comment:
Be careful, this will basically break the installation of the software in casa-distro.